Microsoft.HealthVault/ItemTypes/Address.cs

```diff
@@ -94,16 +94,16 @@ public Address(string city, string country, string postalCode, IEnumerable<strin
             Validator.ThrowIfStringNullOrEmpty(country, "country");
             Validator.ThrowIfStringNullOrEmpty(postalCode, "postalCode");
 
-            this.city = city;
-            this.country = country;
-            this.postalCode = postalCode;
+            this._city = city;
+            this._country = country;
+            this._postalCode = postalCode;
 
             foreach (string streetString in street)
             {
-                this.street.Add(streetString);
+                this._street.Add(streetString);
```

Comment:
Fixed.
  I had gotten partway into fixing issues from removing the this.'s by adding some of them back, when I realized that updating their names would fix it.  So I went through and changed the names but forgot to re-remove the this. references.